docs/guides/_common/ory-cli.mdx

```diff
@@ -1,18 +1,21 @@
-The Ory CLI includes useful functionality to manage your Ory Cloud Project. But
-that is not why we require it in this guide!
+Ory CLI provides a convenient way to configure and manage Ory Cloud projects. Additionally, the CLI 
+contains Ory Proxy - a reverse proxy that rewrites cookies to match the domain your application is currently on. 
```

Comment:
can we now also make the CLI description unified in places such as :  https://docs-gzre1hl01-ory.vercel.app/docs/guides/cli/installation ?

Reply:
I wouldn't do that. The CLI desc. in this snippet is very condensed and tailored to fit this specific context. The dedicated section has more "breathing room" so the descriptions look different.